common/lib/xmodule/xmodule/library_root_xblock.py

```diff
@@ -31,6 +32,12 @@ class LibraryRoot(XBlock):
         help=_("Enter the names of the advanced components to use in your library."),
         scope=Scope.settings
     )
+    show_children_previews = Boolean(
+        display_name=_("Hide children preview"),
```

Comment:
You don't need to translate these because they are never shown to the user. We prefer not to translate strings users won't see, as it adds unnecessary burden to the translators. If this were to be translated, a translator comment would be needed for "Hide children preview", which doesn't make sense on its own.